garage/config_personal_template.py

```diff
@@ -45,3 +45,7 @@
 AWS_ACCESS_KEY = os.environ.get("AWS_ACCESS_KEY", "<insert aws key>")
 
 AWS_ACCESS_SECRET = os.environ.get("AWS_ACCESS_SECRET", "<insert aws secret>")
+
+VICON_TOPICS = ['vicon/vicon_object/iser_cube']
```

Comment:
doesn't apply to all garage users (this is in contrib) so you can remove this file. Just document in your README that they need to add lines to their config.